spec/support/constants.rb

```diff
@@ -6,8 +6,8 @@
 BEEF_TEST_DIR = "/tmp/beef-test/"
 
 # General constants
-ATTACK_DOMAIN = "127.0.0.1"
```

Comment:
Why was this change necessary? Has everything that depends on it been changed?

Reply:
@jcrew99 - it helped prevent a BrowserStack exception, that occasionally occurred when using browserstack_local gem tunnelling + then setting your domain as 127.0.0.1 instead of localhost.

It's the same thing. There was no dependencies that I could identify that require ATTACK_DOMAIN/VICTIM_DOMAIN to be the exact way that they were. They just both need to refer to the local domain, and be different. Please see the documentation @ https://github.com/beefproject/beef/wiki/BeEF-Testing.

It can be changed back with the way we are now handling exceptions, although I do not anticipate it will cause any issues, so I don't think it's worth delaying the merge by another 6 hours.